bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/ShellyBindingConstants.java

```diff
@@ -87,9 +87,9 @@ public class ShellyBindingConstants {
             THING_TYPE_SHELLYPLUSWALLDISPLAY, //
 
             // Shelly Plus Mini
-            THING_TYPE_SHELLYMINI1, //
```

Comment:
let's keep it as is and clean-up with the next PR bringing also Mini G3

Reply:
Thing type id is `shelly1mini`, so it seems inconsistent to use a different name here when the sole purpose of this constant is to reference the thing type.